stable/rabbitmq/values-production.yaml

```diff
@@ -35,6 +35,18 @@ image:
 ## does your cluster have rbac enabled? assume yes by default
 rbacEnabled: true
 
+
+## rabbitmq should be initialized one by one when building cluster for the first time.
+## Therefore, the default value of podManagementPolicy is 'OrderedReady'
+## Once the rabbitmq participates in the cluster, it waits for a response from another 
+## rabbitMQ in the same cluster at reboot, except the last rabbitmq of the same cluster.
+## If the cluster exits gracefully, you do not need to change the podManagementPolicy 
+## because the first rabbitmq of the statefulset always will be last of the cluster.
+## However if the last rabbitmq of the cluster is not the first rabbitmq due to a failure, 
+## you must change podManagementPolicy to 'Parallel'.
+## ref : https://www.rabbitmq.com/clustering.html#restarting
+podManagementPolicy: OrderedReady
```

Comment:
You used both `rabbitmq` and `rabbitMQ`. I'd use `RabbitMQ` for consistency. 

Please also follow the structure below for comments (include an empty comment before the parameter):

```
## Paramater description
## ref: https://xxxx
##
foo: bar

Reply:
Thanks for comments. I fixed it